components/resources/brave_components_resources.grd

```diff
@@ -40,6 +40,11 @@
         <include name="IDR_WALLET_PANEL_HTML" file="../brave_wallet_ui/panel/wallet_panel.html" type="BINDATA" />
       </if>
 
+      <!-- WebUI vpn resources -->
+      <if expr="not is_android">
```

Comment:
I guess this should instead be `enable_brave_vpn` to match up with the GN conditionals